app/workers/downstream_draft_worker.rb

```diff
@@ -6,8 +6,9 @@ class DownstreamDraftWorker
   include PerformAsyncInQueue
 
   sidekiq_options queue: HIGH_QUEUE,
-                  unique: :until_executing,
-                  unique_args: :uniq_args
+                  lock: :until_executing,
+                  unique_args: :uniq_args,
+                  on_conflict: :log
```

Comment:
Did this log before? If not, I can see having this info might be good for debugability but I have a slight concern that it could be an overwhelming quantity so may need some testing.

Reply:
No, this was added here. I've removed it from the commit since we can add it back in if we need to do some debugging.